cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -175,4 +175,16 @@ default_replica_num=1
 # If the number of connections created for a node exceeds  `max_client_pernode_permember_number`,
 # we need to wait so much time for other connections to be released until timeout,
 # or a new connection will be created.
-# wait_client_timeout_ms=5000
\ No newline at end of file
+#wait_client_timeout_ms=5000
+
+# If the path is not updated frequently, you can enable cache to update the path every 1s
+# improving the query performance by 14%
+enable_query_paths_cache=false
+
+# When enable_query_paths_cache is true, this parameter effective.
```

Comment:
grammer error? `this parameter effective`

Reply:
done